archive/generic-core-terms.md

```diff
@@ -0,0 +1,46 @@
+# Introduction
+
+Security is core to our values, and we value the input of hackers acting in good-faith to help us maintain a high standard for the security and privacy for our users. This includes encouraging responsible vulnerability research and disclosure. This policy sets out our definition of good-faith in the context of finding and reporting vulnerabilities, as well as what you can expect from us in return.
+
+# Expectations
+
+When working with us according to this policy, you can expect us to:
+
+- Work with you to understand and validate your report, including a timely initial response to the submission;
+- Work to remediate discovered vulnerabilities in a timely manner; and
+- Recognize your contribution to improving our security if you are the first to report a unique vulnerability, and your report triggers a code or configuration change.
+
+# Scope
+
+# Out-of-Scope
+
+# Rewards
+
+# Disclosure Policy
+
+# Ground Rules
+
+To encourage vulnerability research and to avoid any confusion between legitimate research and malicious attack, we ask that you attempt, in good faith, to:
+
+- Play by the rules. This includes following this policy any other relevant agreements;
+- Report any vulnerability you’ve discovered promptly;
+- Avoid violating the privacy of others, disrupting our systems, destroying data, and/or harming user experience;
+- Use only the Official Channels to discuss vulnerability information with us;
+- Handle the confidentiality of details of any discovered vulnerabilities according to our Disclosure Policy;
+- Perform testing only on in-scope systems, and respect systems and activities which are out-of-scope;
+- If a vulnerability provides unintended access to data: Limit the amount of data you access to the minimum required for effectively demonstrating a Proof of Concept; and cease testing and submit a report immediately if you encounter any user data during testing, such as Personally Identifiable Information (PII), Personal Healthcare Information (PHI), credit card data, or proprietary information;
+- You should only interact with test accounts you own or with explicit permission from the account holder; and
+- Do not engage in extortion.
```

Comment:
Suggested edit:

- Do not engage in extortion _or phishing. Do not misrepresent yourself to be an employee or government official_.

Reply:
Going to nix "employee" from this on account of confused deputy attacks and the like. The government official line feels adequate imo.